code/modules/clothing/under/under.dm

```diff
@@ -64,9 +64,10 @@
 		ATTACHMENT_SLOT_UNIFORM_TIE,
 		ATTACHMENT_SLOT_BADGE,
 	)
-	///Typepath list of allowed attachment types.
+	///Typepath list of uniform variants.
 	var/list/adjustment_variants = list(
 		"Down" = "_d",
+		"Half" = "_h",
```

Comment:
This is adding a halve version of sleeves to ALL the uniforms. Unless you have sprites for them all, you should overwrite on the specific ones you have sprites for.